configure.ac

```diff
@@ -1451,19 +1427,19 @@ TS_CHECK_BROTLI
 TS_CHECK_LUAJIT
 
 # Check for optional quiche library
+enable_quic=no
 TS_CHECK_QUICHE
 if test "${has_quiche}" = "1"; then
 if test "$openssl_is_boringssl" = "1" ; then
 enable_quic=yes
-## Doing these again for Quiche
-AM_CONDITIONAL([ENABLE_QUIC], [test "x$enable_quic" = "xyes"])
-TS_ARG_ENABLE_VAR([use], [quic])
-AC_SUBST(use_quic)
-AC_SUBST(has_quiche)
 else
   AC_MSG_ERROR([Use of BoringSSL is required if Quiche is used.])
 fi
 fi
+TS_ARG_ENABLE_VAR([use], [quic])
+AC_SUBST(use_quic)
+AC_SUBST(has_quiche)
```

Comment:
I thought I can remove it, and I still think we should remove it, but I kept it for now. Because there are files that are used for the both native impl and quiche impl (e.g. iocore/net/QUICNet.cc), and those files `#if TS_HAS_QUICHE `.